recipes/docker.rb

```diff
@@ -127,8 +127,12 @@
             })
 end
 
+service_name='docker'
 # Start the docker deamon
-service 'docker' do
+service service_name do
   action [:enable, :restart]
 end
 
+kagent_config service_name do
```

Comment:
in `docker_registry.rb` you say "hops::docker recipe runs during the install phase and it might run before kagent::install" won't it be a problem to call `kagent_config` here?

Reply:
no because the issue when kagent::install hasn't run, is that `kagent_config` will not find the kagent config file and fail. 
`systemd_reload` does not touch any kagent files